config/default.json

```diff
@@ -12,5 +12,6 @@
 		},
 		"timeout": "30000"
 	},
-	"testsecret": "secret"
+	"testsecret": "secret",
+	"redis": "redis://localhost:6379"

```

Comment:
Redis sollte für localhost und evtl. für test nur optional setzbar sein. 
Daher die Middleware darf auch nur im Fall das es gesetzt ist, an die app gehangen werden.
In Memory als default Fall, wäre für localhost noch schöner. Aber ist zumindest als adapter bei feathers nicht vorgesehen, evtl. andere Lösung?